delegates.txt

```diff
@@ -72,6 +72,7 @@ Claude Pache (CPE)
 Conrad Watt (CWT)
 Conrad Watts (CWS)
 Corey Frang (CF)
+Cuili (CL)
```

Comment:
notes shorthands must all be 3 characters; 2-letter ones predate this requirement.

Reply:
thanks, i  have already dealt with the problem